Cannot find module 'none'

## Search terms

typedoc --plugin none

## Expected Behavior

No crash.

## Actual Behavior

Error: Error: Cannot find module 'none'

## Steps to reproduce the bug

```sh
$ npm add typedoc@beta typescript
$ npx typedoc --plugin none
Error: The plugin none could not be loaded.
Error: Error: Cannot find module 'none'
Require stack:
- /Users/leviticus/cp/node_modules/typedoc/dist/lib/utils/plugins.js
- /Users/leviticus/cp/node_modules/typedoc/dist/lib/utils/index.js
- /Users/leviticus/cp/node_modules/typedoc/dist/lib/models/reflections/project.js
- /Users/leviticus/cp/node_modules/typedoc/dist/lib/models/reflections/index.js
- /Users/leviticus/cp/node_modules/typedoc/dist/lib/models/index.js
- /Users/leviticus/cp/node_modules/typedoc/dist/lib/converter/context.js
- /Users/leviticus/cp/node_modules/typedoc/dist/lib/converter/index.js
- /Users/leviticus/cp/node_modules/typedoc/dist/lib/application.js
- /Users/leviticus/cp/node_modules/typedoc/dist/index.js
- /Users/leviticus/cp/node_modules/typedoc/bin/typedoc
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:885:15)
    at Function.Module._load (internal/modules/cjs/loader.js:730:27)
    at Module.require (internal/modules/cjs/loader.js:957:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.loadPlugins (/Users/leviticus/cp/node_modules/typedoc/dist/lib/utils/plugins.js:10:30)
    at Application.bootstrap (/Users/leviticus/cp/node_modules/typedoc/dist/lib/application.js:156:17)
    at Object.<anonymous> (/Users/leviticus/cp/node_modules/typedoc/bin/typedoc:25:5)
    at Module._compile (internal/modules/cjs/loader.js:1068:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1097:10)
    at Module.load (internal/modules/cjs/loader.js:933:32)
```


## Environment

```json
{
  "dependencies": {
    "typedoc": "^0.21.0-beta.2",
    "typescript": "^4.3.2"
  }
}
```

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "cie.js",
-  "version": "1.0.25",
+  "version": "1.0.26",
   "main": "dist/main.js",
   "types": "dist/main.d.ts",
   "description": "Node wrapper around CIE.sh.",

--- a/package.json
+++ b/package.json
@@ -37,7 +37,7 @@
     "ts-jest": "^27.0.0",
     "ts-loader": "^9.0.0",
     "ts-node": "^10.0.0",
-    "typedoc": "^0.20.34",
+    "typedoc": "^0.21.0",
     "typedoc-plugin-cname": "^1.0.1",
     "typescript": "^4.2.3"
   },

